### ISSUE TITLE
Tensorboard information

### ISSUE THREAD

**Issue by cepLI** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#911]
There was a question about **the overtraining of the model**. Here is the training data of the model (_screen[1..3].png_). Is the underlined place (_screen1.png_) an overtraining point or did I misunderstand something? So the question arises if smoothing is set to 0.999 (_screen2.png_) - it is not clear at all that the model is initially incorrectly trained. Also in the last screenshot (_screen3.png_), the strange behavior of the training is broken, is this a mistake?

![screen1](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/140259609/cc2f87eb-2cc3-4962-ade8-de1c1aea1317)
![screen2](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/140259609/d890a168-cf7e-439a-b451-7c4fe7ff9b41)
![screen3](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/140259609/b249acd3-0e72-4840-8547-96d5af01272c)

**Comment by ms903x1** [comment]
If you terminate the training without reaching the next checkpoint's save epoch, then when you continue training, the program can only continue training from the previous saved checkpoint, and the loss curve recorded by Tensorboard will experience this kind of backtracking, which generally has no impact on the training.

**Comment by ms903x1** [comment]
For the issue of incomplete display of Tensorboard's loss curve, it is only a matter of scaling level. You can try setting the smoothness to 0, double-click the loss curve box you want to view, reset the scaling level, and then adjust the smoothness to 0.6, which may be more convenient to view.
